### PR TITLE
Download notebook data

### PR DESCRIPTION
**Pull request by jslavin**
Changed Deconvolve_XRT_images notebook so that instead of using package data, the data is downloaded from the VSO. This should shrink the size of XRTpy as a package.

### CODE REVIEW

**Comment by joyvelasquez** [comment]
Perhaps trying a simple pip install would do the trick for this failing script?
> python -m pip install lxml

File: docs/notebooks/data_analysis/Deconvolving_XRT_images.ipynb [hunk 1]
```diff
@@ -31,7 +31,7 @@
    "id": "6d3ff603",
    "metadata": {},
    "source": [
-    "Deconvolution is ordinarily used when wanting to remove the blurring around sharp objects or features caused by the telescope's point spread function. Here we show an example of its use for an XRT image taken during the transit of Venus in 2012. This data is included in the test data used by XRTpy."
+    "Deconvolution is ordinarily used when wanting to remove the blurring around sharp objects or features caused by the telescope's point spread function. Here we show an example of its use for an XRT image taken during the transit of Venus in 2012. We download this data from the VSO using methods in Sunpy."
```
Comment (namurphy) [inline docs/notebooks/data_analysis/Deconvolving_XRT_images.ipynb:34]:
```suggestion
    "Deconvolution is ordinarily used when wanting to remove the blurring around sharp objects or features caused by the telescope's point spread function. Here we show an example of its use for an XRT image taken during the transit of Venus in 2012. We download this data from the VSO using methods in SunPy."
```

File: requirements/docs.txt [hunk 2]
```diff
@@ -22,3 +23,4 @@ sphinx_automodapi
 sphinx_rtd_theme >= 1.2
 sphinxcontrib-bibtex >= 2.5
```
Comment (namurphy) [inline requirements/docs.txt:24]:
```suggestion
sphinxcontrib-bibtex >= 2.5
sunpy[net]
```

File: requirements/docs.txt [hunk 3]
```diff
@@ -22,3 +23,4 @@ sphinx_automodapi
...
 towncrier >= 22.12
+zeep
```
Comment (namurphy) [inline requirements/docs.txt:26]:
```suggestion
towncrier >= 22.12
```

File: requirements/docs.txt [hunk 4]
```diff
@@ -8,6 +8,7 @@ docutils >= 0.18
 ipykernel
 ipython
 jinja2 != 3.1
+lxml
```
Comment (namurphy) [inline requirements/docs.txt:11]:
```suggestion
```

**Pull request merged**